javascript/javascript1/week2/FlightBooking.js

```diff
@@ -0,0 +1,19 @@
+function getFullname(firstname,surname) {
+    return firstname,surname;
+}
+console.log("saloumeh","sarabi");
```

Comment:
You are printing ("saloumeh","sarabi") directly to console. But that was not the task :) You need to understand how functions and parameters work in JavaScript and how you can call them.
For example: 
`function printHello(message) {
    console.log(message);
};
printHello("Hello world!!!");`
Try to check some examples online or from your class materials.

Reply:
Hi thanks for all the tips . yes now I can see my mistakes :/. I will redo my homework :).